planning/behavior_path_planner/README.md

```diff
@@ -195,54 +196,153 @@ The Pull Over module is activated when goal is in the shoulder lane. Ego-vehicle
 
 - Pull over request condition
 
-  - The goal is in shoulder lane
-  - The distance from ego-vehicle to the destination is long enough for pull over
-    - The distance required for a pull over is determined by the sum of the straight distance before pull over, the straight distance after pull over, and the minimum distance required for pull over
+  - The goal is in shoulder lane and the ego-vehicle is in road lane.
+  - The distance between the goal and ego-vehicle is somewhat close.
+    - It is shorter than `requet_length`(default: < `100m`).
```

Comment:
```suggestion
    - It is shorter than `request_length`(default: < `100m`).
```

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/1578/commits/0e7de761312528ac65945ae0e8292ed67ff146a3